code/modules/projectiles/guns/projectile/battle_rifle/boltgun.dm

```diff
@@ -13,7 +13,7 @@
 	caliber = CAL_LRIFLE
 	fire_delay = 8
 	damage_multiplier = 1.4
-	style_damage_multiplier = 3
+	style_damage_multiplier = 5
```

Comment:
Will cause them to do 65 style damage, which I feel is excessive. A multiplier of just 3 is enough to prevent dodge from dodging more than two bullets, or three if the person is doing multiple stylish actions.

~Moreover, the marshall lever gun would do 200 style damage instead of the 120 it does with a multiplier of 3.~

Reply:
Yes, bolt guns are slow guns, and those 3 bullets you need to actually land on a target that moves with 0 slowdown.
Just to begin actually damage it, if it won't run away to regen style, like a sane human being would.